src/components/research/secondText.tsx

```diff
@@ -0,0 +1,9 @@
+const SecondaryText = () => {
+  return (
+    <p className="align-center text-center font-seasons text-[30px]">
```

Comment:
```suggestion
    <p className="text-center font-seasons text-[30px]">
```
Use flex justify-center and don't hardcode a text value, just use a tailwind class that looks close enough for now. we'll worry about responsiveness later